site/content/blogs/release-v121.en.md

```diff
@@ -28,12 +28,59 @@ Copilot v1.21 brings several new features and improvements:
 ## CloudFront Integration
 
 ## Configure Environment Security Group
+You can now configure your environment security group rules through environment manifest. 
 
+Sample security group rules template inside environment manifest is given below.
+
+```yaml
+network:
+  vpc:
+    security_group:
+      ingress:
+        - ip_protocol: tcp
+          ports: 0-65535
+          cidr: 0.0.0.0/0
+      egress:
+        - ip_protocol: tcp
+          ports: 80
+          cidr: 0.0.0.0/0
+```
+[See detailed walkthrough](../docs/manifest/environment.en.md#http-public-access-logs)
```

Comment:
Access logs aren't the right thing to link to here. Is there a detailed explanation of the security group you meant to link to? Otherwise I think there should be a small explanation here in the blog.

Reply:
oops I will change this.